helpers/ayah/convert_quran_words_ar_csv.py

```diff
@@ -0,0 +1,58 @@
+'''Convert quran_words_ar.csv to django fixtures file'''
+import json
+from os import path
+import csv
+from collections import defaultdict
+
+
+def structure_surah_dict(data):
+    '''Parse raw csv data and build structured dict'''
+    surah_dict = defaultdict(lambda: defaultdict(list))
+    for surah, ayah, position, word in data:
+        surah_dict[surah][ayah].append(word)
+    return surah_dict
+
+
+def structure_ayah_json(surah_dict):
+    '''Form import ready ayah list json from structured dict'''
+    list_of_ayahs = []
+    for surah, ayahs in surah_dict.items():
+        for ayah, words in ayahs.items():
+            element = {
+                "model": "quran.Ayah",
+                "fields": {
+                    "surah": surah,
+                    "ayah": ayah,
+                    "text": words
```

Comment:
I thought it should be something like `','.join(words)`. Otherwise, you will store square brackets and this is probably not what you intended.